spec/controllers/tube_creation_controller_spec.rb

```diff
@@ -43,7 +43,7 @@
         it 'creates a tube' do
           get :new,
               params: {
-                limber_tube_id: parent_uuid,
+                limber_plate_id: parent_uuid,
```

Comment:
not following, why have these 3 been reversed? was this a bug?

Reply:
Yeah I believe the logic was incorrect here. 'From a plate parent create a tube' So the parent_uuid should be coming from a plate.